build-system/tasks/storybook/env/amp/register.js

```diff
@@ -1,8 +1,3 @@
-import {addons} from '@storybook/addons';
-
 /**
- * Register the AMP Storybook decorator addon
  */
-export function register() {
-  addons.register('amp/storybook', () => {});
-}
+export function register() {}
```

Comment:
Is this an empty override? Any way to remove the file?

Reply:
Yeah it can probably be removed